plugin/pkg/admission/podpreset/admission.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	annotationPrefix = "podpreset.admission.kubernetes.io"
+	exclusionSuffix  = "PodPresetOptOut"
```

Comment:
@derekwaynecarr we can definitely change it - but I believe that the reason we chose the current format is so we can record that multiple pod presets acted on a pod.  What you have suggested doesn't seem to work for multiple pod presets.